lib/dor/services/client.rb

```diff
@@ -88,6 +88,7 @@ def url
         @url || raise(Error, 'url has not yet been configured')
       end
 
+      # rubocop:disable  Metrics/AbcSize
```

Comment:
I don't understand why this suddenly needed to be added since you didn't change anything else in the file

Reply:
Me neither.